sockeye/__init__.py

```diff
@@ -11,4 +11,4 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-__version__ = '1.1.0'
+__version__ = '1.1.1'
```

Comment:
we don't do full semantic versioning (a la http://semver.org/), but the lowest version number should still  be reserved for backwards-compatible changes. Just like with the backwards-incompatible change to the config change we should bump at least the second number if we don't make the change backwards compatible. That said, I'd rather just make the change backwards compatible, in order to not invalidate trained models.